docs/GettingStarted.md

```diff
@@ -99,6 +99,18 @@ src
 │       └── # proto definitions
 ```
 
+To get Gradle to pick up the generated files at compile time, the directory needs to be added as an additional source directory
+to the Gradle config. This can be done by adding the following to `build.gradle`:
+
+```gradle
+sourceSets {
+    main {
+        java {
+            srcDir 'src/main/generated/main/java'
```

Comment:
Did you need to do this to get it working? I don't have this in my `build.gradle` my understanding was that the protobuf plugin was handling this.

Reply:
I actually haven't tried locally (running from *tabula rasa*). However, there was another person who had an issue, and this to their `build.gradle` fixed things. I suppose I could give it a shot on my own.

Reply:
Okay, when I actually tried it, I found that (1) the classpath was actually set up by the protobuf plugin (so I was able to compile the example code) but (2) I needed to add this snippet anyway to make the IDE discover that these files existed. I've updated the example accordingly.